src/Spd.Presentation.Licensing/Controllers/BizLicensingController.cs

```diff
@@ -203,5 +203,32 @@ public async Task<BizLicAppCommandResponse> SubmitBusinessLicenceApplication([Fr
 
             return await _mediator.Send(new BizLicAppSubmitCommand(bizUpsertRequest), ct);
         }
+
+        /// <summary>
+        /// Get biz brand image from its documentId
+        /// </summary>
+        /// <param name="documentId"></param>
+        /// <returns></returns>
+        [Route("api/business-licence-application/brand-image/{documentId}")]
+        [Authorize(Policy = "OnlyBceid")]
+        [HttpGet]
+        public async Task<FileStreamResult> GetBrandImage([FromRoute][Required] Guid documentId, CancellationToken ct)
```

Comment:
is 'documentId' the same as 'documentUrlId' ?

Reply:
yes